### ISSUE TITLE
Manifest Version Deprecation

### ISSUE THREAD

**Issue by iKryptonic** [erickutcher/httpdownloader#209]
![image](https://user-images.githubusercontent.com/23004970/166509249-5780233b-e724-40a9-b1cd-e39612b727d4.png)
Chrome is throwing a deprecation warning for the declared manifest version. 

[Here](https://developer.chrome.com/blog/mv2-transition/) is the linked article on this deprecation. 

The solution seems to be updating to manifest v3 and adhering to any constraints imposed by this migration.

**Comment by erickutcher** [comment]
Yeah I've been working on that on and off. It's on my to-do list.

**Comment by erickutcher** [comment]
It looks like I have 7 months to get this working. Hopefully Google will fix their issues with it too. The extension, right now, will only work on Chrome (and it's derivatives) version 100 and above. They haven't implemented some necessary APIs in the stable releases yet.

If you'd like to try an alpha version of the extension, then you can use this: [HTTP_Downloader_Chrome_Extension_m3.zip](https://github.com/erickutcher/httpdownloader/files/8617860/HTTP_Downloader_Chrome_Extension_m3.zip)

**Comment by iKryptonic** [comment]
I'll use this in the meantime; Thanks for offering your stuff for free, and good luck on the updates / project maintenance!

**Comment by iKryptonic** [comment]
My apologies, I'll leave this open so you can resolve the issue with a commit if you so choose in the future.

**Comment by erickutcher** [comment]
The latest version of the Chrome extension now supports Manifest V3. You can download it here: https://github.com/erickutcher/httpdownloader/releases/tag/CE_v1.0.2.4
